tron/commands/display.py

```diff
@@ -384,9 +385,6 @@ def store_data(self, data):
         self.data = data['runs']
         self.job_run = data
 
-    def rows(self):
```

Comment:
why remove this?

Reply:
by removing that rows method, the actionruns object can use the rows method of the super class, which has sorting support.